helm-chart/flink-operator/values.yaml

```diff
@@ -24,7 +24,7 @@ rbac:
 
 # The defination of the operator image
 operatorImage:
-  name: ghcr.io/spotify/flink-operator:v0.3.3
+  name: ghcr.io/spotify/flink-operator:latest
```

Comment:
Ideally, we want to keep this tied to a release... But since we don't have this in place yet ... It's ok to have it default to the latest for now.

Reply:
yeah, it's better to have a stable/release tag in the future.